README.md

```diff
@@ -1,23 +1,75 @@
-Python courses for the scientific researcher
-============================================
+# Python courses for the scientific researcher
 
 This repository contains several courses for the benefit of scientific researchers,
-particularly in the fields of oceanography and meteorology.
+particularly in the fields of oceanography and meteorology.  
+All the content is fully open-source and freely downloadable.
 
-There are currently four courses:
+These courses are presented as interactive self-learning tutorials,
+contained in ["Jupyter notebooks"](https://jupyter.org/index.html).
+The user will run the notebook interactively, and will also edit it and run
+their own code, particularly to complete the set learning exercises.
 
-### An introduction to numpy
-3.5 hours &mdash; depends on a basic Python background\
-https://mybinder.org/v2/gh/SciTools/courses/master?filepath=course_content%2Fextra_courses%2Fnumpy_intro.ipynb
+The most flexible learning method is to download this repository and run it in a
+Python environment with the correct dependencies.  
+A suitable environment can be produced with
+[conda](https://conda.io/projects/conda/en/latest/user-guide/getting-started.html)
```

Comment:
I'm not sure about this link; it doesn't say anything about miniconda or link to it anywhere, so it's probably a bit confusing for those who are new to conda (given that you later say that miniconda is all you need).

I'd say this should be either a link directly to the miniconda download page or to some miniconda instructions.

Reply:
Ok, I'll look again for something better.
It's tricky, as they mostly promote use of anaconda, so "just" miniconda is a bit of a poor relation.
"https://docs.conda.io/en/latest/miniconda.html" may be a better link.